src/metabase/driver.clj

```diff
@@ -1042,3 +1042,18 @@
   {:added "0.48.0", :arglists '([driver database & args])}
   dispatch-on-initialized-driver
   :hierarchy #'hierarchy)
+
+(defmulti field-values-compatible?
+  "Checks whether some field is eligible for field values scan.
+
+   Problem is that some databases are unable to handle our field value scan query for some fields. Eg. Druid JDBC
```

Comment:
Sorry to be super nitpicky but docstrings should get indented two spaces https://guide.clojure.style/#docstring-indentation

Reply:
This is bit unfortunate, because vsc does this style of indentation by default and to this point I've found no way of changing that :) Actually, good to know that this is also important, I'll try to find a way how to make the editor behave. But with regards to this specific remark -- the multimethod was removed.

Reply:
Honestly it's not really super important but Emacs at least corrects this for me automatically so by sticking to a consistent indentation style we will avoid pointless whitespace changes in PRs.

Didn't realize VS Code is being annyoing, maybe it's time to revisit getting `cljfmt` working again